orchestration/hca_orchestration/solids/create_snapshot.py

```diff
@@ -10,6 +11,7 @@
 
 
 @solid(
+    config_schema=Field(Permissive({"validate_snapshot_name": Field(bool, default_value=True, is_required=False)})),
```

Comment:
I am adding the option to disable snapshot name validation here as the source dataset name in our testing produces datasets that do not pass our validation regex. I would like those datasets to continue not passing the regex validation so they aren't inadvertently picked up by external automation keying on that naming convention.